.evergreen.yml

```diff
@@ -263,21 +265,25 @@ tasks:
       - name: release_debian
         variant: debian
       - name: release_macos
-        variant: darwin
+        variant: darwin-release
     commands:
       - func: checkout
       - func: install
       - func: release_publish
 
 # Need to run builds for every possible build variant.
 buildvariants:
-  - name: darwin
-    display_name: "MacOS Mohave"
+  - name: darwin-release
+    display_name: "MacOS Mojave (codesign machine)"
     run_on: macos-1014-codesign
+    tasks:
+      - name: release_macos
+  - name: darwin
+    display_name: "MacOS Mojave"
+    run_on: macos-1014
     tasks:
       - name: check
       - name: test
-      - name: release_macos
```

Comment:
The changes here are mostly to move work off the codesign machine – there’s only one of them versus 70 other `macos-1014` hosts, and we’re already hogging it a lot.

This also helps with parallelism, because we don’t want to have to wait for that single machine to become available and process `check` and `test` before we can proceed with the release, even for other platforms

Reply:
Actually, I had to undo this because it messed with the `BUILD_VARIANT` variable to have two of these in there … so it’s not this straightforward and probably best kept out of here